lib/main.dart

```diff
@@ -444,6 +451,49 @@ class _MainPageWidgetState extends State<MainPageWidget> {
     }
   }
 
+  /// Detect barcodes from multiple still images
+  Future<void> _detectBarcodesOnImages() async {
+    if (!await checkLicenseStatus(context)) {
+      return;
+    }
+    try {
+      List<Uri> lstUris = List.empty(growable: true);
```

Comment:
`List.empty(growable: true)` -> `<Uri>[] `

This syntax is More familiar for flutter developer